cloud_blobstore/__init__.py

```diff
@@ -1,6 +1,64 @@
 import typing
 
 
+class PagedIter(typing.Iterable[str]):
```

Comment:
@ttung Please check my use of typing for PagedIter, as I have some lingering confusion.

Reply:
👍